source/adios2/ADIOSConfig.h.in

```diff
@@ -50,4 +50,7 @@
 /* CMake Option: ADIOS_USE_Python=ON */
 #cmakedefine ADIOS2_HAVE_PYTHON
 
+/* CMake Option: ADIOS_USE_Fortran=ON */
+#cmakedefine ADIOS2_HAVE_Fortran
```

Comment:
Use all caps for the define

Reply:
Just curious, I see in other CMake files we use ADIOS2_HAVE_Fortran, is the uppercase/lowercase handled by cmake?